geode-core/src/main/resources/org/apache/geode/internal/sanctioned-geode-management-serializables.txt

```diff
@@ -36,5 +36,7 @@ org/apache/geode/management/api/RealizationResult,false,memberName:java/lang/Str
 org/apache/geode/management/configuration/MemberConfig,false,id:java/lang/String
 org/apache/geode/management/internal/cli/domain/ClassName,true,1,className:java/lang/String,initProperties:java/util/Properties
 org/apache/geode/management/runtime/CacheServerInfo,true,1,bindAddress:java/lang/String,isRunning:boolean,maxConnections:int,maxThreads:int,port:int
-org/apache/geode/management/runtime/MemberInformation,true,1,cacheServerList:java/util/List,cacheXmlFilePath:java/lang/String,clientCount:int,cpuUsage:double,groups:java/lang/String,heapUsage:long,host:java/lang/String,hostedRegions:java/util/Set,httpServiceBindAddress:java/lang/String,httpServicePort:int,id:java/lang/String,initHeapSize:long,isCoordinator:boolean,isSecured:boolean,isServer:boolean,locatorPort:int,locators:java/lang/String,logFilePath:java/lang/String,maxHeapSize:long,name:java/lang/String,offHeapMemorySize:java/lang/String,processId:int,serverBindAddress:java/lang/String,statArchiveFilePath:java/lang/String,status:java/lang/String,webSSL:boolean,workingDirPath:java/lang/String
-org/apache/geode/management/runtime/RuntimeRegionInfo,false,entryCount:long: new class
\ No newline at end of file
+org/apache/geode/management/runtime/GatewayReceiverInfo,false,bindAddress:java/lang/String,hostnameForSenders:java/lang/String,port:int,running:boolean
+org/apache/geode/management/runtime/MemberInformation,true,1,cacheServerList:java/util/List,cacheXmlFilePath:java/lang/String,clientCount:int,cpuUsage:double,groups:java/lang/String,heapUsage:long,host:java/lang/String,hostedRegions:java/util/Set,httpServiceBindAddress:java/lang/String,httpServicePort:int,id:java/lang/String,initHeapSize:long,isCoordinator:boolean,isSecured:boolean,isServer:boolean,locatorPort:int,locators:java/lang/String,logFilePath:java/lang/String,maxHeapSize:long,offHeapMemorySize:java/lang/String,processId:int,serverBindAddress:java/lang/String,statArchiveFilePath:java/lang/String,status:java/lang/String,webSSL:boolean,workingDirPath:java/lang/String
```

Comment:
Normally we use this type of file to detect incompatible changes with a previous versions serializable format. Will any of the classes in this file actually be serialized to disk or are all of these just for messaging?
Are these classes just serialized between the locator and the cluster members?
How will the serialization work during a rolling upgrade of a cluster? https://gemfire.docs.pivotal.io/96/gemfire/getting_started/upgrade/upgrade_rolling.html

Reply:
only scenario serialization will occur is for function execution